exercises/practice/error-handling/src/test/java/ErrorHandlingTest.java

```diff
@@ -15,105 +14,83 @@ public class ErrorHandlingTest {
 
     @Test
     public void testThrowIllegalArgumentException() {
-        assertThrows(
-            IllegalArgumentException.class,
-            errorHandling::handleErrorByThrowingIllegalArgumentException);
+        assertThatExceptionOfType(Exception.class).
```

Comment:
Please move the `.` at the end to the beginning of the next line, consistent with the formatting in the other methods.